docs/content/manual/manual.yml

```diff
@@ -3075,13 +3075,29 @@ sections:
           to invoke jq with the -n command-line option, otherwise
           the first entity will be lost.
 
-      - title: "`debug`"
+      - title: "`debug`", "`debug(msgs)`"
```

Comment:
Actually, the CI (on your fork) is red because this is invalid YAML syntax, so the man page cannot even be generated; not only because jq.1.prebuilt is not up to date ;) 

https://github.com/pkoppstein/jq/actions/runs/5538611539/jobs/10108730569

It should be: <code>"&#96;debug&#96;, &#96;debug(msgs)&#96;"</code>